packages/constants/src/tokens.ts

```diff
@@ -49,3 +58,25 @@ export const getTokenAddress = (chainId: CHAINS, token: TOKENS): string => {
 
   return address;
 };
+
+export const getL2TokenAddress = (chainId: CHAINS, token: TOKENS): string => {
+  const tokens = L2_TOKENS_BY_NETWORK[chainId];
+  invariant(tokens, 'L2 chain is not supported');
+
+  const address = tokens[token];
+  invariant(address, 'L2 token is not supported');
+
+  return address;
+};
+
+export const getTokenAddress = (chainId: CHAINS, token: TOKENS): string => {
+  if (token === TOKENS.LDO) {
+    const _chainId =
+      chainId === CHAINS.OptimismSepolia ? CHAINS.Sepolia : chainId;
+    return getL1TokenAddress(_chainId, token);
+  } else {
+    return chainId === CHAINS.OptimismSepolia
```

Comment:
We are going to have here multiple chains. Optimism and then  possibly other l2. So  let's adopt a solution to match multple l2 networks.